packages/payment-integration-api/src/hosted-form/hosted-form-options.ts

```diff
@@ -40,6 +43,11 @@ interface HostedStoredCardFieldOptionsMap {
     [HostedFieldType.CardNumberVerification]?: HostedStoredCardFieldOptions;
 }
 
+type BlueSnapDirectHostedCardFieldOptionsMap = Pick<
+    HostedCardFieldOptionsMap,
+    HostedFieldType.CardName
+>;
```

Comment:
Same here. We want to keep `payment-integration-api` independent of payment method-specific knowledge.

Reply:
Please, see my previous comment.